_campaign-2019/sean-killeen.md

```diff
@@ -23,14 +27,17 @@ I believe that if we're going to serve the various members of our community, we
 * Companies who are "consumer-first" by design
 * Community members who may be a bit demanding / entitled around how OSS works for them
 * Developers who are highly skepitcal of Microsoft but are intrigued by .NET
+* Under-represented community members
 * Microsoft maintainers in the OSS world who often receive incredible amounts of scrutiny and pushback
 * Maintainers of stagnant but popular OSS Projects
 
-And so on (I anticipate many more). By thinking about these different groups, considering their reactions, and confirming with people who identify with these personas or types, we can find the biggest impact, confirm the changes we make will have the intended effect, and communicate more clearly to various stakeholders. I want to us to know our community more fully.
+...and so on (I anticipate many more). By thinking about these different groups, considering their reactions, and confirming with people who identify with these personas or types, we can find the biggest impact, confirm the changes we make will have the intended effect, and communicate more clearly to various stakeholders. I want to us to know our community more fully.
 
 ### Impact Maps: A Technique to Drive the Foundation's Mission
 
-For these types of organizations, I've been heavily inspired by [Gojko Adzic's book on Impact Mapping](https://www.impactmapping.org/book.html). At the risk of serious over-simplifying, I will sum it up by saying that for me, impact mapping is a way to bring many of your stakeholders along to achieve your goals, though that's not quite the original intended purpose (I'd be happy to speak about this subject at length with anyone). 
+Everyone wants to start with things they'll do, or pet ideas that are based on their perpsective. I want to map our problem space and our stakeholders, so that we are conscious about what actions we're taking and what impact they're supposed to achieve.
```

Comment:
`Everyone wants to start with things they'll do, or pet ideas that are based on their perpsective.`

This line can't be backed up and is more opinion.

Reply:
Also `perspective` is misspelled.